skimage/feature/_hoghistogram.pyx

```diff
@@ -0,0 +1,130 @@
+# cython: cdivision=True
+# cython: boundscheck=False
+# cython: wraparound=False
+
+import numpy as np
+cimport numpy as cnp
+
+cdef float cell_hog(cnp.float64_t[:, :] magnitude,
+        cnp.float64_t[:, :] orientation,
+        float orientation_start, float orientation_end,
+        int cell_columns, int cell_rows,
+        int column_index, int row_index,
+        int size_columns, int size_rows):
+    """Calculation of the cell's HOG value
+
+    Parameters
+    ----------
+    magnitude : ndarray
+        The gradient magnitudes of the pixels.
+    orientation : ndarray
+        Lookup table for orientations.
+    orientation_start : float
+        Orientation range start.
+    orientation_end : float
+        Orientation range end.
+    cell_columns : int
+        Pixels per cell (x).
+    cell_rows : int
+        Pixels per cell (y).
+    column_index : int
+        Block column index.
+    row_index : int
+        Block row index.
+    size_columns : int
+        Number of columns.
+    size_rows : int
+        Number of rows.
+
+    Returns
+    -------
+    total : float
+        The total HOG value.
+    """
+    cdef int cell_column, cell_row
+
+    cdef float total = 0.
+    for cell_row in range(-cell_rows/2, cell_rows/2):
```

Comment:
I haven't actually implemented any of these, just typed that out in GitHub. 

This is why we review! When you're just trying to get tests to work, speed isn't foremost in your mind... now we're putting on the final polish.